plutarch-test/src/Plutarch/ListPropSpec.hs

```diff
@@ -0,0 +1,108 @@
+module Plutarch.ListPropSpec (spec) where
+
+import Hedgehog (Property)
+
+import Plutarch.List (pelemAt, pfind, pmergeSort, preverse, ptimSort)
+import Plutarch.Prelude
+
+import Data.List (find, sort)
+
+import Gen (genList, integerGen)
+import Util (haskPlutEquiv, marshal, viaBothPartial, viaPEq)
+
+import qualified Hedgehog.Gen as Gen
+import qualified Hedgehog.Range as Range
+
+import Plutarch.Test (PlutarchGolden (All), goldens)
+import Test.Syd (Spec, describe, it)
+import Test.Syd.Hedgehog ()
+
+spec :: Spec
+spec = describe "list spec" $ do
```

Comment:
By convention we don't have spaces in spec descriptions that get used as golden filenames. This should be therefore `describe "list"`.